### DISCUSSION TITLE
Ubuntu 24.x

### DISCUSSION THREAD

**Discussion by Niklas-PDA** [microsoft/SysmonForLinux#185] [Q&A] [answered]
Hello SysmonForLinux community,
We are planning to implement Ubuntu 24.x and are wondering if sysmon4Linux will support this new version?
we can not find any installation package supporting this version yet..

Regards
Niklas

**Comment by 1ihor1** [comment]
Hello Niklas, 

It should be the same same package as for 20.x, 22.x, 23.x. As I see we have several Ubuntu 24.x with SysmonForLinux in our environment, works fine.
 
Regards,
Igor

**Reply by Niklas-PDA** [reply]
Nice! thank you for the info Igor..

**Reply by MarioHewardt** [reply, marked as answer]
Hi - those packages should work fine. We will release a package for 24.x when we release the next version.

**Reply by darkbounce** [reply]
When will the next release be? 
I want to automate it in my default install of Ubuntu, and now it is failing when I use the **/$(lsb_release -rs)/** instead of a hardcoded **/23.04/**

Any idea on timeframe? (days / weeks / months / whenever we have time.. )

Thanks in advanced and keep up the good work 👍

**Reply by MarioHewardt** [reply]
Hi - We don't have a firm date yet but hopefully during next quarter (March).

**Comment by bauyrzhansaken-prog** [comment]
Good afternoon! Need help or advice
When installing sysmon on Ubuntu 24.04, the service crashes with error: telemetry failed to start: Raw socket program could not be attached
Host specifications:
AMD64 processor
Secure boot enabled
The disk was encrypted with the built-in Ubuntu utility during installation
I haven't touched anything else in the system, a clean Ubuntu installation is running without any settings. I'm just installing sysmon and getting a service startup error
How do I install sysmon:
wget -q https://packages.microsoft.com/config/ubuntu/$(lsb_release -rs)/packages-microsoft-prod.deb -O packages-microsoft-prod.deb

sudo dpkg -i packages-microsoft-prod.deb

sudo apt-get update
sudo apt-get install sysmonforlinux
sysmon -i config.xml
What could be the error or where did I go wrong?
